contributed_definitions/nyaml/NXbeam_device.yaml

```diff
@@ -31,6 +34,10 @@ NXbeam_device(NXobject):
       all these devices have the group name "second harmonic generation".
       Is used for simplified setup vizualization (or description?).
 
+  device:
+    doc: |
+      This is the name of the NeXus device like a NXdetector, a NXmonochromator or NXsource.

```

Comment:
Consider making this a link, i.e., a reference to a full path of data entry. Something like `/entry/instrument/monochromator`. Then it's clear to which detector this refers,

see [here](https://github.com/FAIRmat-NFDI/nexus_definitions/blob/da544c9be46a0d77d877915f59b3a13b698c0c8f/contributed_definitions/nyaml/NXmpes.yaml#L745) for an example